pkg/yurthub/certificate/hubself/cert_mgr.go

```diff
@@ -258,6 +274,24 @@ func (ycm *yurtHubCertManager) initCaCert() error {
 		clusterCABytes = cluster.CertificateAuthorityData
 	}
 
+	if caExisted {
+		var existedCABytes []byte
+		var err error
+		existedCABytes, err = ioutil.ReadFile(caFile)
+		if err != nil {
+			klog.Infof("could not read existed %s ca cert, %v", caFile, err)
+		}
+		if (bytes.Compare(clusterCABytes, existedCABytes) == 0) {
```

Comment:
when yurthub startup if cloud-edge network disconnected, so yurthub can not get `clusterCABytes` from kube-apiserver, leading to yurthub startup failure. so we can not use `CABytes` to verify whether remove certificates or not, and maybe we need to use `bootstrap token` from `yurthub parameter --join-token` and `/var/lib/yurthub/bootstrap-hub.conf` to verify.

Reply:
ok. Let me do bootstrap token way.